src/sass/extends/_card-and-image.scss

```diff
@@ -1,10 +1,12 @@
 .uds-card-and-image {
   background-position: top;
+  background-size: cover;
```

Comment:
- Small fix for a "component" which should probably have never been implemented as a separate thing in the Bootstrap 4 library.
- OK to merge. It doesn't exist as a block within UDS-WordPress as of right now.